graphql/typeDefs.ts

```diff
@@ -13,6 +13,7 @@ export default gql`
     submissions(lessonId: Int!): [Submission!]
     alerts: [Alert!]!
     getPreviousSubmissions(challengeId: Int!, userId: Int!): [Submission!]
+    exerciseSubmissions: [ExerciseSubmission!]!
```

Comment:
This `exerciseSubmissions` query gets all of the exercise submissions for the currently-logged-in user. Maybe a better place to put this would be on the `User` type. This might also be better if we need to grab other users' exercise submissions if we have some feature that requires that.

We might also want to add an `exerciseSubmissions` field to the `Exercise` type in the future as well if we want to add a feature that requires getting all the submissions for an exercise.